src/main/java/seedu/address/logic/commands/AddPersonalTaskCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.personal.PersonalSchedule;
+import seedu.address.model.personal.PersonalTask;
+
+/**
+ * Adds a personal task into the schedule.
+ */
+public class AddPersonalTaskCommand extends UndoableCommand {
+    public static final String COMMAND_WORD = "addTask";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a personal task into the schedule. "
+            + "Parameters: "
+            + "DATE(dd/mm/yyyy) " + "(space) "
+            + "START TIME(hh:mm) " + "(space) "
+            + "DURATION(HhMm) " + "(space) "
+            + "REMARK";
+
+    public static final String MESSAGE_SUCCESS = "New personal task added.";
+
+    private final PersonalTask toAdd;
+
+    /**
+     * Creates an AddPersonalTaskCommand to add the specified {@code Task}.
+     */
+    public AddPersonalTaskCommand(PersonalTask task) {
+        requireNonNull(task);
+        toAdd = task;
```

Comment:
Maybe can have something which shows the schedule that was added (Similar to when adding a person). Like that user can know if they got accidentally add wrongly or not

Reply:
In addition, will the schedule be saved? i.e. when I open the application again, is my schedule there